site/docs/4.1/utilities/text.md

```diff
@@ -85,8 +85,10 @@ Quickly change the weight (boldness) of text or italicize text.
 
 {% capture example %}
 <p class="font-weight-bold">Bold text.</p>
+<p class="font-weight-bolder">Relative bolder than the parent element.</p>
 <p class="font-weight-normal">Normal weight text.</p>
 <p class="font-weight-light">Light weight text.</p>
+<p class="font-weight-lighter">Relative lighter than the parent element.</p>
```

Comment:
Maybe we should add a note that this depends on the system font and the effect might not be visible on some systems.

For example on Windows with Firefox I get:

![2018-11-16_10-33-45](https://user-images.githubusercontent.com/349621/48608420-2e38cb00-e98b-11e8-84a0-a97f7626deb0.jpg)

Reply:
That screenshot is the expected result. That parent is `.bd-example` (the font weight are inherited from `body`)

Reply:
Yeah, that might need a note or, not.

Reply:
Maybe we should just use the same descriptions as in the mozilla docs?
![image](https://user-images.githubusercontent.com/11559216/48645075-d4fd8580-e9e4-11e8-8409-d7fd4d619d54.png)

Reply:
In that case, we should probably link to the MDN doc as a reference.